src/main/java/file/reader/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package file.reader;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> readList = new ArrayList<>();
+
+        try {
+            Scanner scanner = new Scanner(new File(fileName));
+            while (scanner.hasNextLine()) {
+                readList.add(scanner.nextLine());
+            }
+        } catch (FileNotFoundException e) {
```

Comment:
It seems that you are not using the try-with-resources statement to ensure that the Scanner is closed after use. You should wrap the Scanner initialization in a try-with-resources block to automatically close the resource when it's no longer needed.